src/frontends/tensorflow/src/translate_session.cpp

```diff
@@ -242,12 +243,14 @@ void TranslateSession::translate_graph(const ov::frontend::InputModel::Ptr& inpu
                 auto translator = m_translator_map->at(operation_decoder->get_op_type());
                 NodeContext node_context(operation_decoder, ov_inputs, this);
                 ov_outputs = translator(node_context);
-            } catch (const std::exception&) {
-                // continue translation by replacing with FrameworkNode
-                // in case of any failures in translators due to their limitation
+            } catch (const std::exception& ex) {
+                // remember the root-cause about why the translation was failed
+                ov::op::util::FrameworkNodeAttrs attrs;
+                attrs[FrameworkNode::failed_conversion_key] = ex.what();
                 auto fw_node = std::make_shared<FrameworkNode>(operation_decoder,
                                                                ov_inputs,
                                                                operation_place->get_output_ports().size());
+                fw_node->set_attrs(attrs);
                 set_node_name(operation_name, fw_node);
                 ov_outputs = fw_node->outputs();
             }
```

Comment:
That means we are collection status of conversion for `std::exception` only missing other exception origins. Unknown exceptions should be gathered as well and kind of "Unknown exception type" is recorded in FW node. It is especially important for user-defined translators that are our of our control.

Reply:
Fixed.